health-services/individual/src/main/java/org/egov/individual/Constants.java

```diff
@@ -11,5 +11,4 @@ public interface Constants {
     String GET_IDENTIFIER_TYPE = "getIdentifierType";
     String GET_ID = "getId";
 
-    String get();
```

Comment:
There are still some changes in src/main/java in Apicontroller and validator classes. Could you please check if it is due to space?